### ISSUE TITLE
Doesn't work with local kubernetes install

### ISSUE THREAD

**Issue by mhoad** [microsoft/vscode-dapr#142]
Issue Type: <b>Bug</b>

I tried installing dapr on a local microk8s instance but the applications pane in the plugin shows 'The Dapr runtime does not appear to be initialized'. I am assuming this is because it is assuming that I am running the normal solo setup?

Extension version: 0.4.0
VS Code version: Code 1.53.2 (622cb03f7e070a9670c94bae1a45d78d7181fbd4, 2021-02-11T11:48:44.518Z)
OS version: Linux x64 5.8.0-7642-generic

**Comment by philliphoff** [comment]
@mhoad Yes, the extension only supports "standalone" Dapr installations, not those running in a local k8s environment.  That's an interesting idea, though.  How would you expect the application pane to work if it *could* represent Dapr running in a k8s environment?

I'd be a little concerned myself that the application pane end up being *too* k8s aware (and thus being mopre complex) in order to accommodate situations like applications running in different namespaces, etc.

**Comment by theperm** [comment]
Running standalone its not easy to run multiple instances of a service so naturally we just use K8s.

Lets face it, who isnt using K8s ? This extension well have limited appeal if it doesnt.
